pennylane/optimize/rotosolve.py

```diff
@@ -97,8 +141,33 @@ class RotosolveOptimizer:
     separately reconstructing the cost function with respect to each circuit parameter,
     while keeping all other parameters fixed.
 
+    Args:
+        optimizer (str or callable): Optimizer to use for the substeps of Rotosolve
+            that carries out a univariate (i.e. single-parameter) global optimization.
+            It must take a function ``fn`` that maps scalars to scalars and may take optional
+            keyword arguments, and it must return two scalars:
+            The input value ``x_min`` for which ``fn`` is minimal,
+            as well as the minimal value ``y_min=fn(x_min)`` or ``None``.
+            Alternatively, the following optimizers are built-in and can be chosen by
+            passing their name:
+
+            - ``"brute"``: An iterative version of
+              `SciPy's brute force optimizer <https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.brute.html>`_.
+              It evaluates the function at ``Ns`` equidistant points across the range
+              :math:`[-\pi, \pi]` and iteratively refines the range around the point
+              with the smallest cost value for ``num_steps`` times.
+
+            - ``"shgo"``: `SciPy's SHGO optimizer <https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.shgo.html>`_.
```

Comment:
curious if there was any reason for choosing these two?

Reply:
Legacy, I suppose. `brute` is the most straight forwards optimizer, I'd say, and `shgo` caught my attention at some point.
Happy to add others from the scipy library that are particularly good at one-dimensional _global_ optimization.

Reply:
all good! was just curious :)